pages/design/review.md

```diff
@@ -1,5 +1,21 @@
 ---
-title: Auto-WCAG Rule Review Process
+title: Auto-WCAG Rule Design and Review Process
 ---
 
-TODO
\ No newline at end of file
+## Step 1: Creating the proposal in an Issue
+
+Every rule is designed by a team of two or three people. They do their work in a github issue. This means that every activity is represented by an open issue on Github. The first post of an issue contains the proposal they are working on. In subsequent posts, discussions are held, and as they are resolved, the draft at the top of the issue is updated. Once the group is done with the proposal, a pull request is created, and the issue is closed.
+
+## Step 2: Propose a change in a Pull Request
+
+When a proposal in step 1 is done, a pull request is created to update the rule in `_drafts`. This gives the rest of auto-WCAG the opportunity to respond to the proposal. At this point one of three things can happen:
+
+1. Editorial comments can be resolved immediately by updating the pull request. 
+
+2. For any comments that can't be resolved immediately, a new issue is created. Once the review period of the pull request has passed, the pull request is merged in. The team working on this rule continues working on the new issue that was created, going back to step 1.
```

Comment:
"Once the review period of the pull request has passed". We should probably elaborate or change the wording, as the "review period" is not clearly specified, I think.